portability-transfer/src/main/java/org/datatransferproject/transfer/CallableExporter.java

```diff
@@ -34,19 +37,36 @@
   private UUID jobId;
   private AuthData authData;
   private Optional<ExportInformation> exportInformation;
+  private final DtpInternalMetricRecorder metricRecorder;
 
-  public CallableExporter(Provider<Exporter> exporterProvider, UUID jobId, AuthData authData,
-      Optional<ExportInformation> exportInformation) {
+  public CallableExporter(
+      Provider<Exporter> exporterProvider,
+      UUID jobId,
+      AuthData authData,
+      Optional<ExportInformation> exportInformation,
+      DtpInternalMetricRecorder metricRecorder) {
     this.exporterProvider = exporterProvider;
 
     this.jobId = jobId;
     this.authData = authData;
     this.exportInformation = exportInformation;
+    this.metricRecorder = metricRecorder;
   }
 
   @Override
   public ExportResult call() throws Exception {
-    return exporterProvider.get()
-        .export(jobId, authData, exportInformation);
+    boolean success = false;
+    Stopwatch stopwatch = Stopwatch.createStarted();
+    try {
+      ExportResult result =  exporterProvider.get().export(jobId, authData, exportInformation);
+      success = result.getType() != ExportResult.ResultType.ERROR;
```

Comment:
For consistency with the importer, we should probably use ==  ResultType.OK for the success condition here.

Reply:
Ah, for export there are 3 conditions: END, CONTINUE, and ERROR, so I think we have to do != ERROR, or == with an or, so I think the != is cleaner.